uniffi/src/bindings/swift/templates/RustBufferHelper.swift

```diff
@@ -279,3 +282,27 @@ extension Optional: Lowerable where Wrapped: Lowerable {
 }
 
 extension Optional: Serializable where Wrapped: Liftable & Lowerable {}
+
+extension Array: Liftable where Element: Liftable {
+    static func lift(from buf: Reader) throws -> Self {
+        // Int is platform dependent, but so is usize (vec size) on the Rust side!
+        let len = Int(truncatingIfNeeded: try UInt64.lift(from: buf))
```

Comment:
If possible, I think we should throw rather than truncate here; it seems better to fail loudly than risk silently doing the wrong thing. Limiting sequence lengths to `u32` might make this problem go away though.